server/package.json

```diff
@@ -11,7 +11,9 @@
     "cookie-session": "1.3.2",
     "crypto-random-string": "^1.0.0",
     "express": "4.16.0",
+    "fs-extra": "^5.0.0",
     "helmet": "^3.6.1",
+    "jsdom": "https://github.com/nciinc/jsdom/tarball/master/9c957d7b0d60787502979b7566a3c2c91a82aead",
```

Comment:
this is tied to a specific commit while we're working on the fork?

Reply:
Correct.